tests/system/providers/google/cloud/dataflow/example_dataflow_native_python_async_deferrable.py

```diff
@@ -0,0 +1,194 @@
+
+default_args = {
+    "dataflow_default_options": {
+        "tempLocation": GCS_TMP,
+        "stagingLocation": GCS_STAGING,
+    }
+}
+
+with DAG(
+    DAG_ID,
+    default_args=default_args,
+    schedule="@once",
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example", "dataflow"],
+) as dag:
+    create_bucket = GCSCreateBucketOperator(task_id="create_bucket", bucket_name=BUCKET_NAME)
+
+    upload_file = LocalFilesystemToGCSOperator(
+        task_id="upload_file_to_bucket",
+        src=PYTHON_FILE_LOCAL_PATH,
+        dst=PYTHON_FILE_NAME,
+        bucket=BUCKET_NAME,
+    )
+
+    # [START howto_operator_start_python_job_async]
+    start_python_job_async = BeamRunPythonPipelineOperator(
+        task_id="start_python_job_async",
+        runner=BeamRunnerType.DataflowRunner,
+        py_file=GCS_PYTHON_SCRIPT,
+        py_options=[],
+        pipeline_options={
+            "output": GCS_OUTPUT,
+        },
+        py_requirements=["apache-beam[gcp]==2.46.0"],
+        py_interpreter="python3",
+        py_system_site_packages=False,
+        dataflow_config={
+            "job_name": "start_python_job_async",
+            "location": LOCATION,
+            "wait_until_finished": False,
+        },
+    )
+    # [END howto_operator_start_python_job_async]
+
+    # [START howto_sensor_wait_for_job_status]
+    # wait_for_python_job_async_done = DataflowJobStatusSensor(
+    #     task_id="wait_for_python_job_async_done",
+    #     job_id="{{task_instance.xcom_pull('start_python_job_async')['dataflow_job_id']}}",
+    #     expected_statuses={DataflowJobStatus.JOB_STATE_DONE},
+    #     location=LOCATION,
+    # )
+    # [END howto_sensor_wait_for_job_status]
+
+    # [START howto_sensor_wait_for_job_metric]
+    # def check_metric_scalar_gte(metric_name: str, value: int) -> Callable:
+    #     """Check is metric greater than equals to given value."""
+
+    #     def callback(metrics: list[dict]) -> bool:
+    #         dag.log.info("Looking for '%s' >= %d", metric_name, value)
+    #         for metric in metrics:
+    #             context = metric.get("name", {}).get("context", {})
+    #             original_name = context.get("original_name", "")
+    #             tentative = context.get("tentative", "")
+    #             if original_name == "Service-cpu_num_seconds" and not tentative:
+    #                 return metric["scalar"] >= value
+    #         raise AirflowException(f"Metric '{metric_name}' not found in metrics")
+
+    #     return callback
+
+    # wait_for_python_job_async_metric = DataflowJobMetricsSensor(
+    #     task_id="wait_for_python_job_async_metric",
+    #     job_id="{{task_instance.xcom_pull('start_python_job_async')['dataflow_job_id']}}",
+    #     location=LOCATION,
+    #     callback=check_metric_scalar_gte(metric_name="Service-cpu_num_seconds", value=100),
+    #     fail_on_terminal_state=False,
+    # )
+    # [END howto_sensor_wait_for_job_metric]
```

Comment:
The same story here what is the reason for this piece of code?

Reply:
This is one of the questions that I listed in my PR description. It is not clear to me what to do with the example dags while 2 Dataflow operators are still not implemented.
If we decide to not include it into the final PR - then I will remove this.

Reply:
I agree. File removed.